### ISSUE TITLE
Task description

### ISSUE THREAD

**Issue by RicoFio** [RicoFio/IS-master-thesis-template#1]
# Basic OKRs

- [x] The thesis `template.zip` needs to get [this title page](https://github.com/maartenmarx/ThesisTemplate/blob/master/ThesisTemplate/TitlePages/Thesis-Title-Page-DS.tex)
- [x] The thesis template needs to have a title page with the terminology **Internal Supervisor** / **External Supervisor** / **Examiner**. This can come _instead_ of first and second examiner. (This struggle with the correct terminology is the core issue on which there is no consensus.)
- [x] The thesis template needs to _skip_ the terminology of defense and this should be corrected to **submission**. 
- [x] There need to be **line numbers** as in `Format_Thesis_Template.pdf`. (This is a specific wish of Maarten Maarx for feedback purposes and my aim is to facilitate him to get his approval of the template)
- [x] It is important that the line numbers are included in this template as well. 
- [x] There needs to come a similar Latex template for the thesis design to the thesis template without title page and abstract (but for the rest as similar as possible). Introduction, Related Work and Methodology are the same as in the thesis. Risk assessment and Project plan are extra. In the template.
- [x] Possibly create a template for the Exploratory Data Analysis, but I did not yet find the ideal setup. Students interpreted it in a text-rich way, which is not the function of the EDA.
- [x] There should be a template for independent projects as you suggested yourself. I would prefer the template to be like the one of the thesis design and thesis (double column).
- [x] It would be helpful to have a **gantt chart** in the template for the project plan. 

- [x] ~Give an example of how to make sheets of your thesis for a presentation.~ 
- [ ] Publish the template on [Overleaf Gallery](https://it.overleaf.com/learn/how-to/Publishing_templates_and_examples_in_the_Overleaf_Gallery)
- [x] All the examples [here](https://canvas.uva.nl/courses/6056/pages/thesis-design-practical-information#data) are actually not in the right format

## Suggestions for the EDA from @RicoFio 
It might be worth a shot to not force the LateX template on the students for the EDA but rather have them submit a `PDF` version of a  **jupyter notebook**. This would invite them to keep text to a minimum and focus on the actual exploration of the data. It's clear, in my opinion, that if you ask them to submit a thesis-styled document, that they will focus more on the text and the explanation rather than working on the data itself.

# Further requests
- [x] Separate folders for sections and images
- [x] Line numbering
- [x] Use of HyperRef
- [x] Make some bogus sections to show how this works. 

# Preferred workflow
Many students prefer overleaf for their thesis. Hence, we need to facilitate that they stick to these guidelines to keep them organized:

1. Put the URL to your overleaf project in the **title** of your github `README.MD`.
2. Organize your overleaf project well, as follows:
    1. Create directories for 
        1. images
        2. preamble and style files
        3. sections
            - For each section you create a seperate file which you include in `main.tex` at the top level.
            - Use the [subfiles package](https://www.overleaf.com/learn/latex/Multi-file_LaTeX_projects), which simplifies the workflow
        4. bib stuff
3. Top level contains `main.tex`  and the folders, and I guess that is it.

**Comment by RicoFio** [comment]
@lestervanderpluijm whta do you think of my suggestion for the EDA above?

**Comment by lestervanderpluijm** [comment]
@RicoFio I agree. It was never the intention to let students write a lot of text on the EDA. The idea is that you explore the data, get to categorize the type of data you have and possibly reflect on the implications of the distributions (assumptions, etc.).

**Comment by RicoFio** [comment]
@lestervanderpluijm I have just pushed most of the requests here on main and we'll go over the details during our meeting on Monday (15.08.2022). However, I wanted to ask you what you meant with the following:

> Give an example of how to make sheets of your thesis for a presentation.

If you could clarify that today, I can work it in before our meeting :)

**Comment by lestervanderpluijm** [comment]
@RicoFio You can skip idea concerning the sheets template. It will only remove focus from the key templates.

**Comment by RicoFio** [comment]
@lestervanderpluijm cool! We can see what we would like to do for that and possible examples during the meeting then.

If you want to have a peak, I put the pre-compiled files here: `latex_template/compiled-templates`. Maybe you can have a look and we can discuss possible changes during the meeting.
